### ISSUE TITLE
Topo data not loading from gcs

### ISSUE THREAD

**Issue by peterdudfield** [openclimatefix/nowcasting_dataset#197]
**Describe the bug**
Topo data doesnt seem to load from GCS.

**To Reproduce**
I'll add a test 
TODO

**Expected behavior**
Open data ok

**Additional context**
'  rasterio.errors.RasterioIOError: GS_SECRET_ACCESS_KEY+GS_ACCESS_KEY_ID, GS_OAUTH2_REFRESH_TOKEN or GOOGLE_APPLICATION_CREDENTIALS or GS_OAUTH2_PRIVATE_KEY+GS_OAUTH2_CLIENT_EMAIL configuration options and /Users/peterdudfield/.boto not defined
'
- it may be i need to set some env variables

**Comment by peterdudfield** [comment]
@jacobbieker would you mind running the test with your setup and see if it works?

**Comment by jacobbieker** [comment]
Yeah! Sounds good

**Comment by peterdudfield** [comment]
https://github.com/corteva/rioxarray/issues/246

**Comment by JackKelly** [comment]
And, just to check, can you both access other OCF data on Google Cloud from your laptops? Is this problem definitely specific to the topo data?

**Comment by peterdudfield** [comment]
`        filesystem = fsspec.open(self.filename).fs
        if not filesystem.exists(self.filename):
            raise RuntimeError(f"{self.filename} does not exist!")
`

I add this before, and this worked fine

**Comment by jacobbieker** [comment]
> And, just to check, can you both access other OCF data on Google Cloud Storage from your laptops? Is this problem definitely specific to the topo data?

I can download the topo data using `gsutil cp` so I can access it, `gcsfs` just throws that error when trying to access the data when running in the same terminal

**Comment by jacobbieker** [comment]
I guess one way around it could be to download the file locally first, as it is fairly small, then use that cached file for it?

**Comment by JackKelly** [comment]
The issue that @peterdudfield linked to (corteva/rioxarray#246) seems to suggest that, yeah, if we want rioxarray to read data from a cloud storage bucket then we'll have to download the data locally before rioxarray will open it.  I'll ask on that thread.

**Comment by JackKelly** [comment]
Hmm, yeah, I had a quick skim through the docs for [rioxarray](https://corteva.github.io/rioxarray) and for [rasterio](https://rasterio.readthedocs.io/en/stable/topics/reading.html).  I can't see any mention of loading from cloud storage bucket.  The only mention of loading over the network is [loading cloud-optimised GeoTIFFs over HTTP(s)](https://corteva.github.io/rioxarray/html/examples/read-locks.html).  So, yeah, I think we'll need to download locally and read from the local file (and only do this if we're reading from the cloud; not from on-prem).

**Comment by jacobbieker** [comment]
Yeah, from what I saw in the docs, I got the same conclusion. And yeah, I think a little utility like that would probably be the way to go! The start of the download script could create it, then delete the  directory when its done?

**Comment by peterdudfield** [comment]
I'll give something a go
